docs/concepts/pdp/configuration.mdx

```diff
@@ -315,6 +329,70 @@ Default: `False`
 
 Enable FactDB for this PDP. For more information, see the [FactDB documentation](/concepts/pdp/factdb).
 
+#### PDP_CACHE_STORE
+
+Default: `none`
+
+The cache storage type. Options include: none, in-memory, and redis.
+For more information, see the [PDP Cache documentation](/concepts/pdp/overview#pdp-caching-mechanism).
+
+_Added in PDP v0.9.0_
+
+#### PDP_CACHE_TTL
+
+Default: `3600`
+
+Time-to-live in seconds for cached items.
+
+_Added in PDP v0.9.0_
+
+#### PDP_CACHE_MEMORY_CAPACITY
+
+Default: `128`
+
+Memory capacity in MiB for in-memory cache (when using in-memory store).
+
+_Added in PDP v0.9.0_
+
+#### PDP_CACHE_REDIS_URL
```

Comment:
I think it's PDP_CACHE__REDIS__URL ( with double `_` because it's nested twice )

Reply:
Same, it was changed